public/pages/DetectorConfig/components/CodeModal/__tests__/__snapshots__/CodeModal.test.tsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<CodeMOdal /> spec renders the component 1`] = `<div />`;
```

Comment:
Why does the snapshot only have empty `div`?

Reply:
not sure.  Created an issue: https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/issues/58